src/extension/sqlserver/pom.xml

```diff
@@ -24,5 +22,16 @@
       <artifactId>gt-jdbc-sqlserver</artifactId>
     </dependency>
   </dependencies>
-
+  <build>
+    <plugins>
+      <plugin>
```

Comment:
this overrides what is specified in the plugin management section:

https://github.com/geoserver/geoserver/blob/42b62444b6516e7933265644d21a5dabbf16a1b0/src/pom.xml#L1560-L1568

so I wonder why that is (as it overides to use an earlier version of the plugin) -  I think the version element should be removed - but perhaps all of the build of plugin section can be removed as the resources plugin always runs..

Reply:
It can probably, be removed if it is defined else where now.